HTAN.model.csv

```diff
@@ -823,7 +823,7 @@ Publication-associated HTAN Parent Data File ID,HTAN Data File Identifier(s) of
 HTAN Grant ID,HTAN grant number(s) (i.e. CA------ format) associated with the content presented/published.,,,,TRUE,Publication,,,list like
 HTAN Center ID,List of HTAN Center ID(s) associated with the content presented/published.,"HTA1, HTA2, HTA3, HTA4, HTA5, HTA6, HTA7, HTA8, HTA9, HTA10, HTA11, HTA12, HTA13, HTA14",,,TRUE,Publication,,,list like
 Publication Content Type,The type of content presented or published.,"Preprint, Published manuscript, Conference proceedings",,,TRUE,Publication,,,
-DOI,The digital object identifier (DOI) of the content in the form of https://www.doi.org/{doi} to comply with CrossRef DOI display guidelines.,,,,TRUE,Publication,,https://www.doi.org/,url 
+DOI,The digital object identifier (DOI) of the content in the form of https://www.doi.org/{doi} to comply with CrossRef DOI display guidelines.,,,,TRUE,Publication,,https://www.doi.org/,"url "

```

Comment:
Validation rule should not have quote around it or trailing space before end of line

Reply:
Hmm this is weird. I saw these changes but I thought it was probably just a difference in the vis studio csv viewer because these do not show up on my local machine. Just when I push to github. Did you check how it shows on your machine?

Reply:
I will look into why this is happening!